Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION ?= 0.16.0
+VERSION ?= 0.16.1
 REVISION := $(shell git rev-parse --short HEAD;)
```

Comment:
we should probably figure out a way to get rid of these versions everywhere so we can release more easily

Reply:
definitely, should be able to get the tag from git/maybe introduce goreleaser